src/MDMTransitionContext.h

```diff
@@ -83,4 +85,10 @@ NS_SWIFT_NAME(TransitionContext)
  The presentation view controller for this transition.
  */
 @property(nonatomic, strong, readonly, nullable) UIPresentationController *presentationController;
+
+/**
+ Defers execution of the provided work until the completion of the transition.
+ */
+- (void)deferToCompletion:(void (^ _Nonnull)())work;
```

Comment:
Can I call this multiple times? What happens? Please document.

Reply:
Added some documentation.